pkg/attribute/mutableBag.go

```diff
@@ -64,6 +64,14 @@ func GetMutableBag(parent Bag) *MutableBag {
 	return mb
 }
 
+// GetFakeMutableBagForTesting returns a Mutable bag based on the specified map
+// Use this function only for testing purposes.
+func GetFakeMutableBagForTesting(v map[string]interface{}) *MutableBag {
+	m := GetMutableBag(nil)
```

Comment:
There is code to create a mutable bag / or bag replicated in several places in the code base.
The primary reason is that there is no bag constructor that takes the underlying map.

This method is marked for testing use only.